src/Http/Api/Exceptions/HttpClientException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace MultiversX\Http\Exceptions;
+
+use Exception;
+
+class HttpClientException extends Exception
+{
+    //
```

Comment:
Perhaps drop comment - or leave it as it is, if that's more common for empty classes in PHP.

Reply:
Removed✅